common/app/common/RequestLogger.scala

```diff
@@ -34,8 +34,8 @@ case class RequestLoggerFields(request: Option[RequestHeader], response: Option[
       }
       .getOrElse(Map.empty[String, String])
 
-    val allowListedHeaders = allHeadersFields.filterKeys(allowListedHeaderNames.contains)
-    val guardianSpecificHeaders = allHeadersFields.filterKeys(_.toUpperCase.startsWith("X-GU-"))
+    val allowListedHeaders = allHeadersFields.view.filterKeys(allowListedHeaderNames.contains).toMap
```

Comment:
Explanation why we have to add `view`: https://github.com/guardian/ophan/commit/e9ff516ba32b453d770eb0cfed0aa50a1de408ff